scripts/scripts.js

```diff
@@ -127,8 +127,7 @@ const CONFIG = {
     { iframe: 'https://adobe.ideacloud.com' },
   ],
   htmlExclude: [
-    /business\.adobe\.com\/blog\/.*/,
-    /business\.adobe\.com\/(\w\w|(\w\w_\w\w))\/blog\/.*/,
+    /business\.adobe\.com\/(\w\w(_\w\w)?\/)?blog(\/.*)?/,
```

Comment:
Is it safe to assume that blog will not have pages for the `africa`, `mena_en` or `mena_ar` regions? This regex would not pass for those.

Reply:
> Is it safe to assume that blog will not have pages for the `africa`, `mena_en` or `mena_ar` regions? This regex would not pass for those.

the dx blog only supports regions au, uk, de, fr, ko, jp, so it should be okay in this instance!